lib/database.js

```diff
@@ -21,7 +21,7 @@ class Database extends Hookable {
     super()
     this.dir = options.dir || process.cwd()
     this.cwd = options.cwd || process.cwd()
-    this.markdown = new Markdown(options.markdown)
+    this.markdown = new Markdown({ nuxt: options.nuxt, ...options.markdown })
```

Comment:
That would be a breaking change for using it outside of Nuxt (I already do). new Markdown should not depend on `nuxt` instance.

Reply:
👍